src/nodeUtils.ts

```diff
@@ -0,0 +1,68 @@
+const crypto = require('crypto');
+import utils = require('./utils');
+
+/**
+ * Utility functions that rely on Node.js built-in libraries.
+ */
+const nodeUtils = {
+  /**
+   * Node's built in `exec` function sometimes throws outright,
+   * and sometimes has a callback with an error,
+   * depending on the type of error.
+   *
+   * This unifies that interface.
+   */
+  safeExec: (
+    cmd: string,
+    cb: (error: unknown, stdout: string | null) => void
+  ): void => {
+    try {
+      nodeUtils._exec(cmd, cb);
+    } catch (e) {
+      cb(e, null);
+    }
+  },
+
+  // For mocking in tests.
+  _exec: require('child_process').exec,
+
+  /**
+   * https://stackoverflow.com/a/2117523
+   */
+  uuid4: (): string => {
+    // available in: v14.17.x+
+    if (crypto.randomUUID) {
+      return crypto.randomUUID();
+    }
+
+    return utils.uuid4();
+  },
+
+  /**
+   * Secure compare, from https://github.com/freewil/scmp
+   */
+  secureCompare: (a: string, b: string): boolean => {
```

Comment:
This is used in webhooks, how are they getting access to it?

Reply:
Addressed in this commit: https://github.com/stripe/stripe-node/pull/1660/commits/d81bb115ba77c5d3cacb0cd907c879de5d7fc3e4

I added a reference to the Stripe instance in `Webhooks`, from which it can access `Stripe._utils.secureCompare` (also updated the call to `uuid4` in `RequestSender.ts` because it had a similar issue)